src/CMakeLists.txt

```diff
@@ -242,6 +247,18 @@ if(ENABLE_SWIFT)
             ${CMAKE_CURRENT_BINARY_DIR}/swift/Dispatch.swiftmodule
             ${CMAKE_CURRENT_BINARY_DIR}/swift/Dispatch.swiftdoc
           DESTINATION
-            "${INSTALL_TARGET_DIR}/${SWIFT_HOST_ARCH}")
+            ${INSTALL_TARGET_DIR}/${swift_arch})
+
+  if(BUILD_SHARED_LIBS)
+    set(library_kind SHARED)
+  else()
+    set(library_kind STATIC)
+  endif()
+  set(swiftDispatch_OUTPUT_FILE
+      ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_${library_kind}_LIBRARY_PREFIX}swiftDispatch${CMAKE_${library_kind}_LIBRARY_SUFFIX})
+  install(FILES
```

Comment:
This is the reason why I wrote in the review of the other commit that it would be better to just have this be taken care of by add_swift_target. It would ensure that these sorts of things are in sync.

Reply:
Ah, I see.  Yes, adding an install option is reasonable.